nsca_client/__manifest__.py

```diff
@@ -1,24 +1,23 @@
-# -*- coding: utf-8 -*-
 # © 2015 ABF OSIELL <http://osiell.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "NSCA Client",
     "summary": "Send passive alerts to monitor your Odoo application.",
-    "version": "8.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Tools",
-    "website": "http://osiell.com/",
+    "website": "http://github.com/oca/server-tools",
```

Comment:
OCA (don't know if there is an aggregation somewhere on the `website` key, but it could matter I think).